deploy/examples/cluster-test.yaml

```diff
@@ -17,6 +17,8 @@ spec:
   cephVersion:
     image: quay.io/ceph/ceph:v18
     allowUnsupported: true
+  skipUpgradeChecks: false
+  upgradeOSDRequiresHealthyPGs: false
```

Comment:
We try to keep `cluster-test.yaml` as slim as possible. Since these are both defaults and unlikely to need modified for test clusters, I think removing these is best.